rakelib/rswag.rake

```diff
@@ -2,6 +2,22 @@
 
 require 'fileutils'
 
+APPEALS_API_DOCS_DIR = 'modules/appeals_api/spec/docs/v2'
+APPEALS_API_SECTION_SLUGS = Dir["#{APPEALS_API_DOCS_DIR}/*.rb"]
+                            .map { |file_name| file_name.split('/').last.gsub(/_spec.rb$/, '') }
```

Comment:
Since (if I understand correctly) each of the five sections corresponded to an existing single spec file, I was able to add all of the sections at once.